Cargo.toml

```
@@ -1,8 +1,9 @@
 [package]
 name = "bytesize"
-description = "an utility for human-readable bytes representations"
-version = "1.4.0-dev"
-authors = ["Hyunsik Choi <hyunsik.choi@gmail.com>"]
+description = "An utility for human-readable bytes representations. Forked from https://github.com/hyunsik/bytesize ."
+version = "2.0.0"
```

lets hold off on a version bump for right now

```suggestion
version = "1.4.0-dev"
```